client/highlevel-rest/build.gradle

```diff
@@ -36,26 +31,30 @@ dependencies {
   testCompile "org.hamcrest:hamcrest-all:${versions.hamcrest}"
 }
 
-dependencyLicenses {
-  // Don't check the client's license. We know it.
-  dependencies = project.configurations.runtime.fileCollection {
-    it.group.startsWith('org.elasticsearch') == false
+task checkNoOtherDependencies {
```

Comment:
Well make the comment very big, and surround the compile deps. Someone who blindly pastes will also blindly edit.